AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.m

```diff
@@ -207,7 +207,14 @@ - (void)removeAuthToken:(nullable NSString *)authToken {
   if (history) {
     MSLogDebug([MSAppCenter logTag], @"Retrieved history state.");
   } else {
```

Comment:
Do not nest `if` statement. You can do `else if` here.

Reply:
no. Line 218 is something extra and common to all branch in the nested `if` and `else`